sdk/search/Azure.Search.Documents/api/Azure.Search.Documents.netstandard2.0.cs

```diff
@@ -401,6 +403,27 @@ public partial class AzureMachineLearningSkill : Azure.Search.Documents.Indexes.
         public System.Uri ScoringUri { get { throw null; } }
         public System.TimeSpan? Timeout { get { throw null; } set { } }
     }
+    public partial class AzureOpenAIEmbeddingSkill : Azure.Search.Documents.Indexes.Models.SearchIndexerSkill
+    {
+        public AzureOpenAIEmbeddingSkill(System.Collections.Generic.IEnumerable<Azure.Search.Documents.Indexes.Models.InputFieldMappingEntry> inputs, System.Collections.Generic.IEnumerable<Azure.Search.Documents.Indexes.Models.OutputFieldMappingEntry> outputs) { }
+        public string ApiKey { get { throw null; } set { } }
+        public Azure.Search.Documents.Indexes.Models.SearchIndexerDataIdentity AuthIdentity { get { throw null; } set { } }
+        public string DeploymentId { get { throw null; } set { } }
+        public string ResourceUri { get { throw null; } set { } }
+    }
+    public partial class AzureOpenAIParameters
+    {
+        public AzureOpenAIParameters() { }
+        public string ApiKey { get { throw null; } set { } }
+        public Azure.Search.Documents.Indexes.Models.SearchIndexerDataIdentity AuthIdentity { get { throw null; } set { } }
```

Comment:
.NET doesn't typically abbreviate or shorten like this. Perhaps a client teaspoon or autorest transform to make this "AuthenticationIdentity" (or is it "AuthorizationIdentity")?

Reply:
In the upcoming preview release, we are using the same names as those in the Swagger spec. However, we are planning to rename these types in the next release.